catalyst/data/augmentor.py

```diff
@@ -6,22 +6,42 @@ class Augmentor:
     Augmentation abstraction to use with data dictionaries.
     """
     def __init__(
-        self, dict_key: str, augment_fn: Callable, default_kwargs: Dict = None
+        self,
+        dict_key: str,
+        augment_fn: Callable,
+        input_key: str = None,
+        output_key: str = None,
+        default_kwargs: Dict = None,
```

Comment:
I think, you can use `**default_kwargs`

Reply:
I'm not sure I follow you, do you mean that it is better to replace `default_kwargs: Dict = None,` with `**default_kwargs`?

Reply:
yup, I think it's better for user experience to just write `(... output_key=str, some_val=1)` rather than `(... output_key=str, default_kwargs={some_val=1})`